run.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+if [ "$EUID" -ne 0 ]
+  then sudo $0 $@
+  exit
+fi
```

Comment:
I quite don't like this hack. A user with permissions to the `dockerd` daemon will be able to run this without `sudo`, and forcing sudo is not necessary.